app/jobs/assignment_repo/porter_status_job.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class AssignmentRepo
+  class PorterStatusJob < ApplicationJob
+    class ImportInProgress < StandardError; end
+    queue_as :porter_status
+    retry_on Octopoller::TimeoutError, queue: :porter_status
+
+    # rubocop:disable MethodLength
+    def perform(assignment_repo, user)
+      github_repository = assignment_repo.github_repository
+
+      result = Octopoller.poll do # TODO: log errors
```

Comment:
We should probably log any errors that come our way here even if they are about to be swallowed. Do we log errors anywhere else in the app?

Reply:
If you need logging, you can use `Rails.logger`